plugins/scancode-fingerprint/README.md

```diff
@@ -0,0 +1 @@
+A ScanCode scan plugin to generate fingerprints using Simhash algorithm
```

Comment:
Can you add a bit more detail here? Maybe some short background on the algorithm or some links AND how to go about running it from the command line?